### ISSUE TITLE
Anyone else using docker in windows having an issue w/ it binding ports?

### ISSUE THREAD

**Issue by TTVert** [brbeaird/SmartThings-MyQ-Edge#29]
Maybe i'm missing something here but no matter what I do when running the command **docker run -d --name='smartthings-myq-edge' --network=host 'brbeaird/smartthings-myq-edge:latest'** it won't bind to the ports.  If I manually specify ports w/ the -p switch I'm good.  Not a huge deal but curious as to what I may be missing.  I've got a static on my pc running docker and this is the command I run: **docker run --restart=always -d -p 0.0.0.0:8090:8090/tcp -p 0.0.0.0:1900:1900/udp --name='DB' 'brbeaird/smartthings-myq-edge:latest'**    For the sake of argument and to ensure no conflicts even w/ the other container disabled I did try to remove all containers and run the command w/ no luck.

When I specify port mapping:
![image](https://user-images.githubusercontent.com/24374538/231291986-eb486899-89e4-4b72-b32c-c297c739ae99.png)

When I run the --network=host switch:
![image](https://user-images.githubusercontent.com/24374538/231291662-8ae3f8c9-1f02-46a1-b4e9-31d209828dc6.png)

![image](https://user-images.githubusercontent.com/24374538/231291746-58844116-9a18-44f5-9f32-bc3667a104b4.png)

Am I missing something?  

Thanks
Dave

**Comment by TTVert** [comment]
Nobody at all?  Is nobody using docker in windows?

**Comment by brbeaird** [comment]
I have gotten it to work with static IP's with Windows docker, but I just clicked around the GUI - I never messed with command-line. My main server is Unraid, though, so I didn't feel the need to dig too deep into it.

**Comment by TTVert** [comment]
Fair enough, maybe i can pool my brain w/ someone else using this and we can figure it out.  The machine running docker does have a static but if ports always change that's an issue for me so I need to specify ports.  Just not sure why the binding is broken when using your host command.

Dave

**Comment by feartheclown** [comment]
Thanks for this. I had a ton of issues getting the Docker version to work originally and decided to try it out again now that the EXE is not functioning at this time. This is explains why I could never get it working before.

In any case, poking around it looks like it may not work as intended for Windows.
https://forums.docker.com/t/ubuntu-linux-network-host-not-using-host-network/133736
https://docs.docker.com/network/drivers/host/

Might be worth your command being added to the Docker for Desktop install instructions for Windows? I think it might be the missing sauce why I've myself and seen others have issues getting it working and falling back on the EXE.
